server/main.go

```diff
@@ -41,24 +41,49 @@ var (
 	httpPort      = flag.Int("http_port", 8080, "HTTP serving port.")
 	schemaRootDir = flag.String("schema_root_dir", "schema/alpha", "Root directory for the predefined schemas.")
 
+	mlmdDBType					 = flag.String("mlmd_db_type", "mysql", "Database type to use when creating MLMD instance. Supported options: in-memory, mysql, sqlite")
 	mlmdDBName           = flag.String("mlmd_db_name", "mlmetadata", "Database name to use when creating MLMD instance.")
 	mySQLServiceHost     = flag.String("mysql_service_host", "localhost", "MySQL Service Hostname.")
 	mySQLServicePort     = flag.Uint("mysql_service_port", 3306, "MySQL Service Port.")
 	mySQLServiceUser     = flag.String("mysql_service_user", "root", "MySQL Service Username.")
 	mySQLServicePassword = flag.String("mysql_service_password", "", "MySQL Service Password.")
+	sqliteFilenameUri		 = flag.String("sqlite_filename_uri", "mlmetadata", "Sqlite Filename URI")
```

Comment:
Maybe run goformat on the file before submitting.

Reply:
Done. Could you re-lgtm?